src/components/PhotoUserCollab/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { Container, PhotoUser } from "./styles";
+import ImageUserDefault from "../../img/userdefault.jpg";

```

Comment:
@MarcoBrunoBR Is importing the image this way correct?

Reply:
Yes, it's correct!